nservicebus/messaging/discard-old-messages.md

```diff
@@ -57,10 +57,12 @@ TimeToBeReceived relies on the transport infrastructure to discard expired messa
 
 MSMQ continuously checks the TimeToBeReceived of all queued messages. As soon as the message has expired, it is removed from the queue, and disk space gets reclaimed. 
 
-NOTE: MSMQ enforces single TimeToBeReceived value for all messages in a transaction. If multiple messages enlist in a single transaction, then TimeToBeReceived from the first message will be used for all messages, leading to potentially, unintentional message expiration. To prevent message loss, `TimeToBeReceived` is not supported for endpoints with [transaction mode](/transports/transactions.md) `TransportTransactionMode.AtomicSendsWithReceive` or `Transaction Scope (Distributed Transaction)`.
+NOTE: MSMQ enforces a single TimeToBeReceived value for all messages in a transaction. To prevent message loss, `TimeToBeReceived` is not supported out of the box for endpoints with [transaction mode](/transports/transactions.md) `SendsAtomicWithReceive` or `TransactionScope`. 
```

Comment:
I've been trying to avoid the term "out of the box" in docs in favour of something like "by default". But I can't accurately say why other then it feels too colloquial.